src/lib.rs

```diff
@@ -794,7 +794,7 @@ impl UtcDateTime {
 }
 
 /// Date time associated to a local time type, exprimed in the [proleptic gregorian calendar](https://en.wikipedia.org/wiki/Proleptic_Gregorian_calendar)
-#[derive(Debug, Clone, Eq, PartialEq)]
+#[derive(Debug, Clone, Eq)]
```

Comment:
`Eq` cannot be derived and must be implemented manually since it is not applicable to all fields.

Reply:
Eq is more of a marker, having an empty body, but I made the implementation explicit.